intermediate_notebooks/E2E/mortgage/mortgage_e2e.ipynb

```diff
@@ -629,10 +697,8 @@
     "    'scale_pos_weight':  2,\n",
     "    'min_child_weight':  30,\n",
     "    'tree_method':       'gpu_hist',\n",
-    "    'n_gpus':            1,\n",
-    "    'distributed_dask':  True,\n",
     "    'loss':              'ls',\n",
-    "    'objective':         'gpu:reg:linear',\n",
+    "    'objective':         'reg:squarederror',\n",
```

Comment:
`gpu:objective_func` no longer seems to work. Open to suggestion for using any other objective function

Reply:
@robocopnixon can you review these xgboost param changes?

Reply:
This should be `binary:logistic` for this problem.